core/client/fs/src/main/java/alluxio/client/file/cache/CacheManager.java

```diff
@@ -208,6 +209,16 @@ default int get(PageId pageId, int pageOffset, int bytesToRead, byte[] buffer,
   int get(PageId pageId, int pageOffset, int bytesToRead, byte[] buffer, int offsetInBuffer,
       CacheContext cacheContext);
 
+  /**
+   * Get page ids by the given file id.
```

Comment:
is it "get page ids" or "get ids of the pages which are cached"?
if later case, shall we called `getCachedPageIdsByFileId`?

Reply:
getCachedPageIdsByFileId looks better, thx!